### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by bsian03**
e44190f onwards, i forgot to reset my branch

### CODE REVIEW

File: src/modules/purge.js [hunk 1]
```diff
@@ -39,5 +39,26 @@ module.exports = bot => {
 	}
   });
 
-  bot.registerCommandAlias('p', 'purge');
+	bot.registerCommandAlias('p', 'purge');
+	
+	addInboxServerCommand('undo', async (msg, args, thread) => {
```
Comment (CoalSephos) [inline src/modules/purge.js:44]: isn't this just the purge command but not admin locked?

**Review by CoalSephos — APPROVED** [review]
this is good - provides redundancy for an unstable dashboard

File: src/modules/purge.js (hunk 1 above)
Comment (bsian03) [reply src/modules/purge.js:44]: Just 1 message

Comment (bsian03) [reply src/modules/purge.js:44]: See L53

Comment (CoalSephos) [reply src/modules/purge.js:44]: yes but you could just do it in quick succession

Comment (CoalSephos) [reply src/modules/purge.js:44]: i also apparently can’t read, i shouldn’t have been doing these while trying to note down stellar classifications - my oopsie

**Pull request merged**